client/src/app/containers/Alert/slice.ts

```diff
@@ -0,0 +1,26 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+import { AlertType } from './types';
+
+export interface Alert {
```

Comment:
:mag: Does this conflate/conflict with the `export` of `client/src/app/containers/Alert/index.tsx`?

Reply:
The `export` from `client/src/app/containers/Alert/index.tsx` is a default export, and it's being exported from a filename which happens to be a bit special in typescript/javascript (`index.tsx`). That means I can do this: `import whateverIWantToCallIt from 'client/src/app/containers/Alert'`. It's not a named export, so you see that I can alias it to whatever I want when I import it.

On the other hand, this export is a named export called `Alert`. I have to import it as: `import { Alert } from 'client/src/app/containers/Alert/slice`.